sdk/textanalytics/Azure.AI.TextAnalytics/src/autorest.md

```diff
@@ -7,7 +7,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    -  https://github.com/Azure/azure-rest-api-specs/blob/bc1d3e6f1d19840375907101b30579c16c8a5d76/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.2/TextAnalytics.json
+    -  $(this-folder)/swagger.json
```

Comment:
I thought swagger was already merged. Could we changed to that one instead of a local copy?

Reply:
It could be in another PR if it makes things easier

Reply:
i ll open a new PR once i am in good shape with this PR.